### PR TITLE
Mongodb on ubuntu updated

### PR DESCRIPTION
**Pull request by SychevIgor**
script required some update:
### Changelog
drop storage account
removed second subnet
updated to production api versions
mongodb 4 script update
current vm generation

### CODE REVIEW

File: mongodb-on-ubuntu/azuredeploy.json [hunk 1]
```diff
@@ -42,17 +36,47 @@
...
     },
     "vmSize": {
       "type": "string",
-      "defaultValue": "Standard_A0",
+      "defaultValue": "Standard_A1_v2",
       "metadata": {
         "description": "VM Size"
-      }
+      },
+      "allowedValues": [
```
Comment (bmoore-msft) [inline mongodb-on-ubuntu/azuredeploy.json]: remove this since it creates a maint problem later

Comment (SychevIgor) [reply mongodb-on-ubuntu/azuredeploy.json]: you mean- allowedValues? or new default value?

Comment (bmoore-msft) [reply mongodb-on-ubuntu/azuredeploy.json]: remove the allowedValues property

Comment (SychevIgor) [reply mongodb-on-ubuntu/azuredeploy.json]: @bmoore-msft done

**Pull request merged**